FWCore/Utilities/interface/RandomNumberGenerator.h

```diff
@@ -153,10 +153,10 @@ namespace edm {
     /// These are the only functions most modules should call.
 
     /// Use this engine in event methods
-    virtual CLHEP::HepRandomEngine& getEngine(StreamID const&) const = 0;
```

Comment:
@Dr15Jones IT was caused by the ModuleIDToEngine needing to return a non-const random number generator engine.